source/Betting_Box_Class.py

```diff
@@ -0,0 +1,44 @@
+import random
+import os
+import Card_Class
+import Betting_Box_Class
+import Dealer_Class
+import Deck_Class
+import Player_Class
+
```

Comment:
I second Tommy's main objections. Also, the test for this would be simply to make sure everything runs and all functionality has been maintained. Just as a reminder, this is why tests are so important. If you need assistance let me know!

Reply:
Sounds good! I was thinking I didn’t need a test for this sprint but it all makes sense now.